src/redux/categories/categories.js

```diff
@@ -0,0 +1,52 @@
+const ADD = 'my-app/categories/ADD';
+const REMOVE = 'my-app/categories/REMOVE';
+const CHECK_STATUS = 'my-app/categories/CHECK_STATUS';
+
+export default function categoryReducer(state = [], action) {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state,
+        {
+          id: action.id,
+        },
+      ];
+    case REMOVE:
+      return state.filter((category) => category.id !== action.id);
+
+    case CHECK_STATUS:
+      return state.map((category) => {
+        if (category.id === action.id) {
+          return {
+            ...category,
+            status: action.status,
+          };
+        }
+        return category;
+      });
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] Also kindly make your reducer to only have the `CHECK_STATUS` action type as it is only the action required according to the project requirement